pom.xml

```diff
@@ -32,7 +39,11 @@
     </properties>
 
     <dependencies>
-      <!-- As document-generator is progressed depedencies will be added -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-hateoas</artifactId>
```

Comment:
What is this being used for?

Reply:
addition of spring boot to the parent pom. required me to add the dependancy.